backend/plan/management/commands/trainrecommender.py

```diff
@@ -338,6 +346,15 @@ def train_recommender(
         ), "If you are training on data from S3, there's no need to supply a local data path"
     if course_data_path is not None:
         assert course_data_path.endswith(".csv"), "Local data path must be .csv"
+    if course_descriptions_path is not None:
+        assert course_descriptions_path.endswith(
+            ".csv"
+        ), "Local course descriptions path must be .csv"
+        assert course_data_path is not None, (
+            "If course_description_path is provided then course_data_path should also"
+            "be provided. Note that the courses should be ordered the same in"
```

Comment:
Looking at `generate_course_vectors_dict`, it seems that the courses don't actually have to be ordered the same in both csvs (the courses in `course_description_path` just have to be a superset of the `courses` in course_data)

Reply:
Good point. That's some old documentation from a previous implementation that I changed. I'll remove that.